src/main/java/pl/javastart/task/Person.java

```diff
@@ -0,0 +1,102 @@
+package pl.javastart.task;
+
+import java.util.Objects;
+
+public class Person {
+
+    private static final int MINIMUM_AGE = 1;
+    private static final int MINIMUM_LETTERS = 2;
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String pesel;
+
+    public Person(String firstName, String lastName, int age, String pesel) {
+        throwExceptionIfNameIsNotCorrect(firstName);
+        throwExceptionIfNameIsNotCorrect(lastName);
+        throwExceptionIfAgeIsNotCorrect(age);
```

Comment:
Użyłbym nazw "validateName()", ale to już kwestia preferencji.

Inna opcja to użyć tutaj setterów, zamiast przypisywać do pól bezpośrednio, ale tak też jest ok